livekit-plugins/livekit-plugins-cartesia/livekit/plugins/cartesia/tts.py

```diff
@@ -200,6 +223,7 @@ async def sentence_stream_task():
         async def input_task():
             async for data in self._input_ch:
                 if isinstance(data, self._FlushSentinel):
+                    self._req_ch.send_nowait(None)
```

Comment:
from a user perspective implementing a tts/stt, this is unclear why this is needed